src/Tmds.Ssh/SshClientSettings.Defaults.cs

```diff
@@ -63,6 +65,40 @@ private static IReadOnlyList<Credential> CreateDefaultCredentials()
         return credentials.AsReadOnly();
     }
 
+    private static List<Name> CreatePreferredEncryptionAlgorithms()
+    {
+        // Prefer AesGcm over ChaCha20Poly when the platform has AES instructions.
```

Comment:
Do we do this as AES with hardware acceleration is faster or is this what OpenSSH does as well?

Reply:
This is for ordering from fastest to slowest.

OpenSSH doesn't try to be smart. It has a fixed order.
The system config files enable the administrator to re-order the algorithms.

The `SshClient` can be created so it uses the OpenSSH config files (and follows their configuration).